requirements/base.txt

```diff
@@ -2,4 +2,4 @@ astropy
 numpy
 pandas
 requests
-sunpy
+sunpy==0.9.0
```

Comment:
Can this be a `<=` instead of an `==` so it places less restrictions on the version number?

Reply:
On it! But I think 0.9.0 works perfectly right now